notebooks/03_evaluate/evaluation.ipynb

```diff
@@ -48,8 +48,8 @@
      "name": "stdout",
```

Comment:
<p>the logloss formula can be -inf, however, I'm not that sure about the sklearn implementation, I think the minimum is 0</p>

 _Reply via <a href='https://app.reviewnb.com/Microsoft/Recommenders/pull/591/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='58'/>

Reply:
You are right, the logloss value should minimum 0. Will fix it.

Reply:
You are right. It should be minimum 0.